src/main/scripts/src/app/workspace/organization/organization-settings.component.ts

```diff
@@ -119,4 +124,8 @@ export class OrganizationSettingsComponent implements OnInit {
       this.organizationDescription.nativeElement.focus();
     }, 50);
   }
+
+  public initialized(): boolean {
+    return this.organization.code != '' && this.organization.name != '';
```

Comment:
`!=` doesn't check for type, `!==` is preffered
(`0 != ''` -> `false`; `0 !== ''` -> `true`)

`return this.organization.code != '' && this.organization.name != '';` ->
`return this.organization.code !== '' && this.organization.name !== '';`

(Also `string != ''` is often redundant and can be replaced with `string` as it checks for empty string too)